CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - `sandbox.sh` now accepts a `--consensus-update-timeout-ms` to modify the `consensus.message_timeout` value in each node's configuration. This can be used to alter multi-node commit latency.
 
+### Changed
+
+- CCF is now a separate project and package per platform (sgx, snp and virtual), rather than the same project and package with a decorated version, to prevent accidental misuse and narrow down dependencies. (#4421).
```

Comment:
I think "separate project" is misleading. Do we mean "a separate CMake project"? And I think we should give examples of the names of the projects and packages. This is a breaking change that's going to affect every C++ application, so should clearly explain what the fix is.